fe/fe-core/src/main/java/com/starrocks/common/util/concurrent/lock/Locker.java

```diff
@@ -267,15 +264,11 @@ public boolean isDbWriteLockHeldByCurrentThread(Database database) {
         }
     }
 
-    private boolean checkExistenceInLock(Database database, boolean isInReadLock) {
+    private boolean checkExistenceInLock(Database database, LockType lockType) {
```

Comment:
suggest to use another name? this name seems have a misunderstanding for asserting the lock is holden by someone,   releaseLockIHolden， releaseLockIfPresent?

Reply:
Seems no matter?  This is only a private method to check locks exists.